code/modules/antagonists/changeling/powers/mutations.dm

```diff
@@ -278,10 +264,10 @@
 	if(isitem(target))
 		var/obj/item/I = target
 		if(!I.anchored)
-			to_chat(firer, "<span class='notice'>You pull [I] towards yourself.</span>")
-			add_attack_logs(src, I, "[src] pulled [I] towards them with a tentacle")
-			H.throw_mode_on()
-			I.throw_at(H, 10, 2)
+			to_chat(firer, "<span class='notice'>You grab [I] with your tentacle.</span>")
+			add_attack_logs(H, I, "[src] grabs [I] with a tentacle")
+			I.forceMove(H.loc)
+			I.attack_hand(H)//The tentacle takes the item back with them and makes them pick it up. No silly throw mode.
```

Comment:
Why not do a regular pickup?

Reply:
good question, will change

Reply:
Attackhand may be the best option, directly calling pickup doesn't seem to work, as well as some items having checks on attackhand instead of pickup for restrictions. If a better solution is found, will switch to it.